app/assets/javascripts/govuk_publishing_components/components/intervention.js

```diff
@@ -0,0 +1,70 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function Intervention ($module) {
+    this.$module = $module
+    this.$closeLink = this.$module.querySelector('.js-dismiss-link')
+    this.$campaignName = this.$module.getAttribute('data-intervention-name')
+    this.$interventionCookie = window.GOVUK.cookie('intervention_campaign')
+    this.$campaignCookie = window.GOVUK.cookie('intervention_campaign') || ''
```

Comment:
campaignCookie and interventionCookie appear to be almost the same variable, is there a reason they couldn't be one?